docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
```

Comment:
Can we differentiate this docker-compose.yml from the `compose.yml` file we already have to make it clearer that this one is for dev?